src/main/java/org/snomed/snowstorm/syndication/services/importers/customversion/loinc/LoincSyndicationService.java

```diff
@@ -36,15 +36,11 @@ protected List<File> fetchTerminologyPackages(SyndicationImportParams params) th
         return singletonList(file.orElseThrow(() -> new ServiceException("Loinc terminology file not found, cannot be imported")));
     }
 
-    /**
-     * Will import the loinc terminology. If a loinc terminology file is already present on the filesystem, it will use it.
-     * Else, it will download the latest version or version @param version if specified
-     */
     @Override
     protected void importTerminology(SyndicationImportParams params, List<File> files) throws IOException, InterruptedException {
         String fileName = files.get(0).getName();
         Process process = new ProcessBuilder(
-                "./hapi-fhir-cli", "upload-terminology",
+                "../../hapi/hapi-fhir-cli", "upload-terminology",
```

Comment:
Oh, you're using hapi-fhir-cli to give you that "update while server is running" functionality aren't you.   That's like a new functional area again.   I'll need to also cross-check the documentation on installing that hapi-fhir-cli.   Hard coded paths like this give me the heebie-jeebies.